### PR TITLE
Fix user not loaded in Sample page

### PR DESCRIPTION
**Pull request by jleblond**
To be merged on both QC and Master.

The other approach to this problem would be to test for store.users.isFetching.
However at the SamplesDetailContent page level, we only access the level store.users.itemsByID, so isFetching is not accessible from there. I thought it is best to simply test for 'user' rather than importing into the mapStateToProps store.users or store.users.isFetching

### CODE REVIEW

File: src/components/samples/SamplesDetailContent.js [hunk 1]
```diff
@@ -209,11 +209,13 @@ const SamplesDetailContent = ({samplesByID, containersByID, individualsByID, use
 };
 
 function renderTimelineLabel(version, usersByID) {
-  const user = usersByID[version.revision.user];
+  const user = usersByID[version.revision.user]
+  const username = user ? user.username : ""  // in case usersByID is still loading asynchronously
+
```
Comment (romgrk) [inline src/components/samples/SamplesDetailContent.js:214]: `const username = user?.username` would be preferable if you're going to default to an empty string (which I dont recommend).

The comment is not necessary if you render a loading indicator, e.g. `const username = user?.username ?? "Loading..."`

**Comment by UlysseFG** [comment]
This seem to be branched from master ... we will cancel this PR and do the changes on the right branch ...

**Comment by jleblond** [comment]
See https://github.com/c3g/freezeman_web/commit/3437676dd032ca2eed9a38168b1092dcfe3beea3

**Comment by romgrk** [comment]
I'm not sure I understand the branching model here, which is the leading branch? Isn't QC supposed to be behind master?

**Pull request closed without merging**